packages/astro/src/core/routing/manifest/create.ts

```diff
@@ -159,6 +159,29 @@ function comparator(a: Item, b: Item) {
 	return a.file < b.file ? -1 : 1;
 }
 
+function injectedRouteToItem(
```

Comment:
This is used to sort injected routes with the same comparer used for user-defined routes